src/infrastructure/selectors/JsonSelectorAdapter.js

```diff
@@ -8,7 +8,16 @@ class JsonSelectorAdapter extends DataSelectorPort {
 
   select(data, path) {
     const json = typeof data === 'object' ? data : JSON.parse(data);
-    const value = jp.value(json, path.substr(1));
+
+    if (!path) {
+      return JSON.stringify(json);
+    }
+
+    const value = jp.value(json, `$${path}`);
+
+    if (!value) {
+      return null;
```

Comment:
How can we avoid returning `null` here and in other selectors?

Reply:
Null is suppose to be here if there is no data selected. Is the expected behavior.

Reply:
I see that. One could argue that returning `null` is a bad practice and maybe it could be avoided somehow if you're refactoring the selectors.